No wordexp(3) on OpenBSD.
XXX check for the wordexp function instead of using G_OS_UNIX + push upstream

Index: src/editor-path.c
--- src/editor-path.c.orig
+++ src/editor-path.c
@@ -26,7 +26,7 @@
 #include <glib/gi18n.h>
 #include <string.h>
 
-#ifdef G_OS_UNIX
+#if defined G_OS_UNIX && !defined(__OpenBSD__)
 # include <unistd.h>
 # include <wordexp.h>
 #endif
@@ -38,7 +38,7 @@
 char *
 _editor_path_expand (const gchar *path)
 {
-#ifdef G_OS_UNIX
+#if defined G_OS_UNIX && !defined(__OpenBSD__)
   wordexp_t state = { 0 };
   char *escaped = NULL;
   char *ret = NULL;
